Failed: ManualTimerExampleSpec

https://jenkins.akka.io:8498/job/akka-nightly-2.12/547/testReport/junit/akka.testkit.typed.scaladsl/ManualTimerExampleSpec/A_timer_must_replace_timer/

```
 java.lang.AssertionError: assertion failed: received unexpected message Tock(2)
	at scala.Predef$.assert(Predef.scala:219)
	at akka.testkit.typed.internal.TestProbeImpl.expectNoMessage_internal(TestProbeImpl.scala:172)
	at akka.testkit.typed.internal.TestProbeImpl.expectNoMessage(TestProbeImpl.scala:162)
	at akka.testkit.typed.scaladsl.ManualTime.$anonfun$expectNoMessageFor$1(ManualTime.scala:55)
	at akka.testkit.typed.scaladsl.ManualTime.$anonfun$expectNoMessageFor$1$adapted(ManualTime.scala:55)
	at akka.testkit.typed.scaladsl.ManualTime$$Lambda$530/2132707865.apply(Unknown Source)
	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:32)
	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:29)
	at scala.collection.mutable.WrappedArray.foreach(WrappedArray.scala:34)
	at akka.testkit.typed.scaladsl.ManualTime.expectNoMessageFor(ManualTime.scala:55)
	at akka.testkit.typed.scaladsl.ManualTimerExampleSpec.$anonfun$new$9(ManualTimerExampleSpec.scala:99)
```

--- a/akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala
+++ b/akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala
@@ -6,7 +6,6 @@ package akka.actor.testkit.typed.scaladsl
 
 //#manual-scheduling-simple
 import akka.actor.typed.scaladsl.Behaviors
-import com.typesafe.config.ConfigFactory
 
 import scala.concurrent.duration._
 

--- a/akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala
+++ b/akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala
@@ -6,6 +6,7 @@ package akka.actor.testkit.typed.scaladsl
 
 //#manual-scheduling-simple
 import akka.actor.typed.scaladsl.Behaviors
+import com.typesafe.config.ConfigFactory
 
 import scala.concurrent.duration._
 
@@ -69,6 +70,7 @@ class ManualTimerExampleSpec extends AbstractActorSpec {
       case class SlowThenBump(nextCount: Int) extends Command
       sealed trait Event
       case class Tock(n: Int) extends Event
+      case object SlowThenBumpAck extends Event
 
       val probe = TestProbe[Event]("evt")
       val interval = 10.millis
@@ -83,21 +85,22 @@ class ManualTimerExampleSpec extends AbstractActorSpec {
             case SlowThenBump(nextCount) ⇒
               manualTime.timePasses(interval)
               timer.startPeriodicTimer("T", Tick(nextCount), interval)
+              probe.ref ! SlowThenBumpAck
               Behaviors.same
           }
         }
       }
 
       val ref = spawn(behavior)
-
       manualTime.timePasses(11.millis)
       probe.expectMessage(Tock(1))
 
-      // next Tock(1) enqueued in mailboxed, but should be discarded because of new timer
+      // next Tock(1) enqueued in mailbox, but should be discarded because of new timer
       ref ! SlowThenBump(2)
-      manualTime.expectNoMessageFor(interval, probe)
+      probe.expectMessage(SlowThenBumpAck)
+      manualTime.expectNoMessageFor(9.millis, probe)
 
-      manualTime.timePasses(interval)
+      manualTime.timePasses(2.millis)
       probe.expectMessage(Tock(2))
     }
 

